files/en-us/learn/server-side/express_nodejs/deployment/index.html

```diff
@@ -154,8 +160,9 @@ <h3 id="Log_appropriately">Log appropriately</h3>
 export DEBUG="author,book"
 </pre>
 
-<div class="note">
-<p><strong>Challenge:</strong> Calls to <code>debug</code> can replace logging you might previously have done using <code>console.log()</code> or <code>console.error()</code>. Replace any <code>console.log()</code> calls in your code with logging via the <a href="https://www.npmjs.com/package/debug">debug </a>module. Turn the logging on and off in your development environment by setting the DEBUG variable and observe the impact this has on logging.</p>
+<div class="notecard note">
+  <h4>Challenge</h4>
+  <p>Calls to <code>debug</code> can replace logging you might previously have done using <code>console.log()</code> or <code>console.error()</code>. Replace any <code>console.log()</code> calls in your code with logging via the <a href="https://www.npmjs.com/package/debug">debug </a>module. Turn the logging on and off in your development environment by setting the DEBUG variable and observe the impact this has on logging.</p>
```

Comment:
```suggestion
  <p>Calls to <code>debug</code> can replace logging you might previously have done using <code>console.log()</code> or <code>console.error()</code>. Replace any <code>console.log()</code> calls in your code with logging via the <a href="https://www.npmjs.com/package/debug">debug</a> module. Turn the logging on and off in your development environment by setting the DEBUG variable and observe the impact this has on logging.</p>
```
Sorry, my OCD over here :sweat_smile: